lib/utils/date.dart

```diff
@@ -39,6 +39,18 @@ class BreezDateUtils {
     return isAfter && isBefore;
   }
 
+  static DateTime? blockDiffToDate({required int blockHeight, required int? expiryBlock}) {
+    if (expiryBlock != null) {
+      int diff = expiryBlock - blockHeight;
+      if (diff > 0) {
+        diff *= 600;
+        final time = DateTime.now();
+        return time.add(Duration(seconds: diff));
+      }
+    }
+    return null;
+  }
+
```

Comment:
Let's make this non-nullable.
```suggestion
  static DateTime blockDiffToDate({required int blockHeight, required int expiryBlock}) {
      int diff = expiryBlock - blockHeight;
      if (diff > 0) {
        diff *= 600;
        final time = DateTime.now();
        return time.add(Duration(seconds: diff));
      }
  }

```

Reply:
I suppose it should return current date if there's no difference.

Reply:
Done